code-pushup.config.ts

```diff
@@ -39,6 +40,10 @@ export default mergeConfigs(
   await lighthouseCoreConfig(
     'https://github.com/code-pushup/cli?tab=readme-ov-file#code-pushup-cli/',
   ),
+  await typescriptPluginConfigNx({
+    tsconfig:
+      'packages/plugin-typescript/mocks/fixtures/basic-setup/tsconfig.json',
```

Comment:
> any idea how we can use it best on our code?

It would make sense to check our packages. I see that only 1 TS project is supported at present, would be useful to support an array. The ESLint and Coverage plugins support this setup, and also include Nx helpers to auto-populate the configurations. But since only 1 is possible for now:

```suggestion
    tsconfig: 'packages/cli/tsconfig.lib.json',
```